Gemfile

```diff
@@ -19,7 +19,7 @@ group :test do
   gem 'rake', '~> 10'
   gem 'rubocop', '~> 0.36.0'
   gem 'simplecov', '~> 0.10'
-  gem 'concurrent-ruby', '~> 0.9'
+  gem 'concurrent-ruby', '~> 1.0'
```

Comment:
Since this is a major version bump, will it affect anything down stream? ChefDK for example? Or is that not an issue here?

Reply:
Not to worried as its only in the test bracket. But with the pining it should not be an issue.